inc/function.inc.php

```diff
@@ -20,7 +20,17 @@ function curl_URL_call($url){
     curl_setopt($ch, CURLOPT_URL, $url);
     curl_setopt($ch, CURLOPT_HEADER, 0);
     curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1); 
-    $output = curl_exec($ch);
-    curl_close($ch);
-    return $output;
+    
+    
+    $retcode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
```

Comment:
There is no need of making an extra variable
if condition can very well be
`if (curl_getinfo($ch, CURLINFO_HTTP_CODE) == 200) {`